### ISSUE TITLE
thank you for your good project,  it works

### ISSUE THREAD

**Issue by viponedream** [PeterDaveHello/tor-socks-proxy#31]
i can use 9150 to connect to the proxy.

but i have some questions
1, about the ports.   9050  9150   9151
    i need 9150  and 9151.

2, in tor browser, i  would select obfs4,  it seems it's the best way to connect to tor for me. 
can i use obfs4 in this docker?

**Comment by PeterDaveHello** [comment]
You can change the published port mapping of the Docker container to use 9150 or 9151 port, but I'm not using obfs4, I suggest changing the issue subject to something more related to the content so that some passengers might come and help.

**Comment by b1oki** [comment]
Yup, It would be nice to be able to use the bridge, probably by specifying the path to a file with a list of bridges.
